modules/openapi-generator/src/main/resources/Java/libraries/apache-httpclient/api.mustache

```diff
@@ -15,6 +15,7 @@ import {{invokerPackage}}.Pair;
 
 {{^fullJavaUtil}}
 import java.util.ArrayList;
+import java.util.Collections;
```

Comment:
make sure import is present when `fullJavaUtil=true`

Reply:
Done.

While testing with `fullJavaUtil=true` I found many other places across several generators where this setting also caused problems. I fixed most of them in my latest commit.

That there aren't many issues and PRs about `fullJavaUtil` problems might indicate that this setting is not used much and you might want to deprecate it?

Reply:
OMG.. probably in that case would be better to revert your last change and leave as it is. Thanks for pointing this out

Reply:
I reverted it. Is there anything else left do be done?